WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -452,7 +452,8 @@ nick_the_fair = {
 		NOT = { age = 50 }
 	}
 	chance = {
-		factor = 1
+		factor = 1.5
+
```

Comment:
It's an attempt to increase the variety of nicknames. The "Sword of X" shouldn't be as common as it is.